src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_COST = 40;
+
+  if (days < 3) {
+    return days * BASE_COST;
+  } else if (days < 7) {
+    return days * BASE_COST - 20;
+  } else {
+    return days * BASE_COST - 50;
```

Comment:
Similarly, avoid using `else` after an `if` with a `return`. Just write the return statement on its own, as it simplifies the logic and readability of the function.